control/tests/statesp_test.py

```diff
@@ -5,9 +5,10 @@
 
 import unittest
 import numpy as np
-from scipy.linalg import eigvals
+from numpy.linalg import solve
+from scipy.linalg import eigvals, block_diag
 from control import matlab
-from control.statesp import StateSpace, _convertToStateSpace
+from control.statesp import StateSpace, _convertToStateSpace,tf2ss
```

Comment:
Though `tf2ss` is added here, it is not used in this module (statesp_test.py). Am I missing something?